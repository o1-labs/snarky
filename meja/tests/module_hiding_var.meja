module A = {
  let x = 15;
};

module B = {
  module A = {
    let y = 20;
  };
};

open B;

let () = ignore (A.y);

/* Expect an error, A should be out of scope. */
let () = ignore (A.x);
