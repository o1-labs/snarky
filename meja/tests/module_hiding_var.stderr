File "tests/module_hiding_var.meja", line 16, characters 17-20:
Error: Unbound name A.x.
